tg_react/settings.py

```diff
@@ -8,10 +8,23 @@
     from django.utils.module_loading import import_by_path as import_string
 
 
+# fields unconditionally excluded from user signup data
+USER_SIGNUP_SKIPPED_FIELDS = {'id', 'is_staff', 'is_superuser', 'is_active', 'date_joined', 'last_login'}
```

Comment:
Actually USER_SIGNUP_SKIPPED_FIELDS have nothing to do with UserDetailsSerializer since they are only used for signup. Also when a user is signing up a new User instance is created, and I assume the id field should not be passed to the constructor.
As for `UserDetailsSerializer.Meta.extra_kwargs`, I added `is_superuser` as read only though it was somewhat out of scope of this PR :)